<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerSessionSettingsConfigWidget</class>
 <widget class="QWidget" name="PerSessionSettingsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Per Session Settings</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="includeMarkersSwitch">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enclose &lt;code&gt;#include&lt;/code&gt; directive into &lt;code&gt;&amp;lt;...&amp;gt;&lt;/code&gt; instead of &lt;code&gt;&amp;quot;...&amp;quot;&lt;/code&gt; for session include dirs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>#include session headers w/ '&lt;'/'&gt;'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useCurrentDirSwitch">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enclose &lt;code&gt;#include&lt;/code&gt; directive into &lt;code&gt;&amp;lt;...&amp;gt;&lt;/code&gt; instead of &lt;code&gt;&amp;quot;...&amp;quot;&lt;/code&gt; for session include dirs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Implicitly use current dir &quot;.&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
